.github/workflows/ci-commit.yml

```diff
@@ -5,3 +5,5 @@ on:
 jobs:
   lint:
     uses: ./.github/workflows/ci-lint.yml
+  gosec:
+    uses: ./.github/workflows/ci-gosec.yml
```

Comment:
do we need this in `ci-pr.yml` (or `ci-default.yml`) to ensure it runs on & blocks merge of PRs?

asking b/c i don't know :)

Reply:
I'll add it to `ci-default.yml` since that workflow is being called in `ci-pr.yml` anyways